src/util/isArrayLike.ts

```diff
@@ -0,0 +1 @@
+export const isArrayLike = ((x: any): boolean => x && typeof x.length === 'number');
```

Comment:
The extra parens around both of these examples confused me.

Also, protip: in Github if you start with ```js  it will format the code block as JavaScript, TypeScript works too, for the most part.